nilearn/decomposition/multi_pca.py

```diff
@@ -163,32 +99,41 @@ class MultiPCA(BaseEstimator, TransformerMixin, CacheMixin):
         The mask of the data. If no mask was given at masker creation, contains
         the automatically computed mask.
 
-    `components_`: 2D numpy array (n_components x n-voxels)
+    `components_`: 2D numpy array (n_components x n-voxels)ca
         Array of masked extracted components. They can be unmasked thanks to
-        the `masker_` attribute.
+        the `_pca_masker_` attribute.
```

Comment:
Same comment as above _pca_masker_ doesn't seem to be used anywhere in the code, maybe the name has changed.